docs/DeveloperGuide.md

```diff
@@ -319,127 +302,36 @@ Given below is an example usage scenario, and an object diagram to show the obje
 Step 1. The user launches the application. The `ReadOnlyAddressBook` will be initialized with the initial address book
 state.
 
-Step 2. The user
-executes `addm m/moduleName l/lectureDetails t/tutorialDetails z/zoomLink a/assignment1 a/assignment2` command to add a
-module with the corresponding details in the address book. The `addm` command
-calls `AddressBookParser#parseCommand()`, which creates a `AddModuleCommandParser`. The `AddModuleCommandParser` then
-tokenizes the user input string and returns an ArgumentMultimap object that maps prefixes to their respective argument
-values.
-
-Methods in `ParserUtil` is are then called to parse each individual object obtained from the ArgumentMultimap using
-their corresponding parsers. Then, a new `module` with the corresponding details is created. A `AddModuleCommand` is
-created, which
-calls `Model#addModule()`, and adds the newly created module to the model object.
+Step 2. The user executes `addm m/CS1101S` command to add a module with the corresponding details in the address book. 
+* The `addm` command calls `AddressBookParser#parseCommand()`, which creates a `AddModuleCommandParser`. 
+* The `AddModuleCommandParser` then tokenizes the user input string and returns an `ArgumentMultimap` object that maps 
+prefixes to their respective argument values. 
+* Methods in `ParserUtil` is are then called to parse each individual object obtained from the `ArgumentMultimap` using
+their corresponding parsers. 
+* Then, a new `Module` with the corresponding details is created. 
+* After creating the `Module`, an `AddModuleCommand` is created, which calls `Model#addModule()`, 
+and adds the newly created module to the model object.
 
 The following object diagram illustrates the above example:
-(to be added)
-[//]: # (![AddModuleObjectDiagram]&#40;images/AddModuleObjectDiagram.png&#41;)
+![AddModuleObjectDiagram](images/AddModuleObjectDiagram.png)
 
 The following sequence diagram shows how the AddModule operation works:
-(to be added)
-[//]: # (![AddModuleSequenceDiagram]&#40;images/AddModuleSequenceDiagram.png&#41;)
-
-
-### \[Proposed\] Undo/redo feature
-
-#### Proposed Implementation
```

Comment:
I think this part can be kept here, we can just add features that are more relevant to SoCompiler